src/lib/zork/freedom-module.ts

```diff
@@ -44,11 +44,38 @@ var PORTS = [9000, 9010, 9020];
 let numOfGetters = 0;
 
 let isMetricsEnabled = false;
+let serverId :string = null;  // Must be set in order to post metrics.
 if (typeof freedom !== 'undefined') {
   const parentFreedomModule = freedom();
   parentFreedomModule.on('setMetricsEnablement', function(newValue) {
     isMetricsEnabled = newValue;
   });
+  parentFreedomModule.on('setServerId', function(newValue) {
+    serverId = newValue;
+  });
+}
+
+function postMetrics(startUtcMs: number, endUtcMs: number, getterInstanceId?: string) {
```

Comment:
Please explain that `startUtcMS` and `endUtcMs` are milliseconds elapsed since the UNIX epoch.